README.md

```diff
@@ -66,6 +66,7 @@ A curated list of Minio community projects inspired by [awesome-go](https://gith
 * [s3git](https://github.com/s3git/s3git) - git for cloud storage. s3git provides distributed version control for data. Create decentralized and versioned repos that scale infinitely to 100s of millions of files. Clone huge PB-scale repos on your local SSD to make changes, commit and push back. Check out [s3 docs](http://rclone.org/s3/) for more information.
 
 ### Cloud Storage Clients
+* [Cloud Explorer] (https://github.com/rusher81572/cloudExplorer) - Cloud Explorer is an open source client written in Java and runs on any OS. It has many unique features such as a a text editor, performance testor, image viewer, search, bucket migrations, music player, IRC client, and much more.
```

Comment:
Please remove the space between [Cloud Explorer] and (the URL). GitHub doesn't recognize this as a hyperlink right now.